homeassistant/components/plugwise/binary_sensor.py

```diff
@@ -33,32 +33,31 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
 
     all_devices = api.get_all_devices()
     for dev_id, device_properties in all_devices.items():
-        if device_properties["class"] != "heater_central":
-            continue
-
-        data = api.get_device_data(dev_id)
-        for binary_sensor, dummy in BINARY_SENSOR_MAP.items():
-            if binary_sensor not in data:
-                continue
-
-            entities.append(
-                PwBinarySensor(
-                    api,
-                    coordinator,
-                    device_properties["name"],
-                    binary_sensor,
-                    dev_id,
-                    device_properties["class"],
+        if device_properties["class"] == "heater_central":
+            data = api.get_device_data(dev_id)
+
+            for binary_sensor, dummy in BINARY_SENSOR_MAP.items():
```

Comment:
seems the use of items() here is redundant since they value isnt used.

Reply:
`binary_sensor` is used in lines 40 and 49, that is why the `.items()` is there.

But, when looking at the code with a fresh eye, we probably could simplify the BINARY_SENSOR_MAP as we are not using the value-part. Is this the thinking behind your comment?

Reply:
What @ctalkington means is we don't have to `dummy` because we only use the keys.

Reply:
Ok clear.